egs2/TEMPLATE/sds1/app.py

```diff
@@ -0,0 +1,797 @@
+import argparse
+import os
+import shutil
+import time
+
+import gradio as gr
+import nltk
+import numpy as np
+import torch
+from huggingface_hub import HfApi
+from pyscripts.utils.dialog_eval.ASR_WER import handle_espnet_ASR_WER
+from pyscripts.utils.dialog_eval.human_feedback import (
+    natural_vote1_last_response,
+    natural_vote2_last_response,
+    natural_vote3_last_response,
+    natural_vote4_last_response,
+    relevant_vote1_last_response,
+    relevant_vote2_last_response,
+    relevant_vote3_last_response,
+    relevant_vote4_last_response,
+)
+from pyscripts.utils.dialog_eval.LLM_Metrics import (
+    DialoGPT_perplexity,
+    bert_score,
+    perplexity,
+    vert,
+)
+from pyscripts.utils.dialog_eval.TTS_intelligibility import (
+    handle_espnet_TTS_intelligibility,
+)
+from pyscripts.utils.dialog_eval.TTS_speech_quality import TTS_psuedomos
+
+from espnet2.sds.espnet_model import ESPnetSDSModelInterface
+
+access_token = os.environ.get("HF_TOKEN")
+ASR_name = None
+LLM_name = None
+TTS_name = None
+ASR_options = []
+LLM_options = []
+TTS_options = []
+Eval_options = []
+upload_to_hub = None
+dialogue_model = None
+
+
+def read_args():
```

Comment:
Maybe `parse_args`? Is there any specific reason on naming this function to be `read_args`?

Reply:
Thanks Masao for the feedback! I have renamed the function based on your suggestion!